src/composables/auth.ts

```diff
@@ -105,29 +122,29 @@ export const useAuth = createCustomScopedComposable(() => {
       return;
     }
     composableInitialized = true;
-    await checkSecureLoginAvailability();
+    await checkBiometricLoginAvailability();
   })();
 
   watch(
```

Comment:
This watcher will be set up every time the composable will be used which is bad. I think this should happen only once per the app instance so I suggest to move it into the "initialized" block.